changelog.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unreleased
 
-_Nothing yet..._
+- react-native-bpk-component-button-link:
+  - Introducing the new react native button link component 
```

Comment:
`React Native` should be capitalised, and the sentence should end in a full stop 🙈 .

Reply:
😅 sorry, I should know this by now

Reply:
We should have a Danger rule